xds/internal/xdsclient/clientimpl.go

```diff
@@ -69,22 +145,57 @@ func (c *clientImpl) BootstrapConfig() *bootstrap.Config {
 	return c.config
 }
 
-// close closes the gRPC connection to the management server.
+// close closes the xDS client and releases all resources.
 func (c *clientImpl) close() {
```

Comment:
I think i asked this above. Will user of the xds client call this or its internally called on some condition?